Makefile

```
@@ -58,6 +58,18 @@ test-e2e: export OASIS_INDEXER_E2E = true
 test-e2e:
 	@$(GO) test -race -coverpkg=./... -coverprofile=coverage.txt -covermode=atomic -v ./tests/e2e
 
+dump-state: oasis-indexer
+	sed -E -i '' 's/query_on_cache_miss: false/query_on_cache_miss: true/g' "tests/api/api-dev.yml"
+	./oasis-indexer --config tests/api/api-dev.yml
+
+# Run the api tests locally, assuming the environment is set up with an oasis-node that is 
+# accessible as specified in the config file.
+test-api: oasis-indexer
+	sed -E -i '' 's/query_on_cache_miss: true/query_on_cache_miss: false/g' "tests/api/api-dev.yml"
+	./oasis-indexer --config tests/api/api-dev.yml
```

```suggestion
	./oasis-indexer --config tests/api/api-dev.yml analyze
```
or else the server part will keep this process running forever

---

thanks, updated!